Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -53,7 +53,8 @@ var SCROLLVIEW_REF = 'listviewscroll';
  * `ListView.DataSource`, populate it with a simple array of data blobs, and
  * instantiate a `ListView` component with that data source and a `renderRow`
  * callback which takes a blob from the data array and returns a renderable
- * component.
+ * component. The ListView will only be able to scroll if the parent's style
+ * has 'flex: '.
```

Comment:
This isn't quite right -- the problem is that ListView won't scroll unless it has a non-zero height. Flex is one way to achieve that, but I think if you specified height: 400, it would scroll too.